templates/dashboards/grafana-dashboard-image-builder-worker-general.configmap.yml

```diff
@@ -9,2587 +9,2736 @@ metadata:
-            "auto_min": "10s",
-            "current": {
-              "selected": false,
-              "text": "6h",
-              "value": "6h"
-            },
-            "hide": 0,
-            "name": "interval",
-            "options": [
-              {
-                "selected": false,
-                "text": "5m",
-                "value": "5m"
-              },
-              {
-                "selected": false,
-                "text": "30m",
-                "value": "30m"
-              },
-              {
-                "selected": false,
-                "text": "1h",
-                "value": "1h"
-              },
-              {
-                "selected": true,
-                "text": "6h",
-                "value": "6h"
-              },
-              {
-                "selected": false,
-                "text": "12h",
-                "value": "12h"
-              },
-              {
-                "selected": false,
-                "text": "1d",
-                "value": "1d"
-              },
-              {
-                "selected": false,
-                "text": "3d",
-                "value": "3d"
-              },
-              {
-                "selected": false,
-                "text": "7d",
-                "value": "7d"
-              }
-            ],
-            "query": "5m,30m,1h,6h,12h,1d,3d,7d",
-            "queryValue": "",
-            "refresh": 2,
-            "skipUrlSync": false,
-            "type": "interval"
-          },
-          {
-            "description": "Compose stability SLO target",
-            "hide": 2,
-            "name": "stability_slo",
-            "query": "0.95",
-            "skipUrlSync": false,
-            "type": "constant"
-          },
-          {
-            "description": "Compose Duration SLO target",
-            "hide": 2,
-            "name": "latency_slo",
-            "query": "0.95",
-            "skipUrlSync": false,
-            "type": "constant"
-          }
-        ]
-      },
-      "time": {
-        "from": "now-28d",
-        "to": "now"
-      },
-      "timepicker": {
-        "refresh_intervals": [
-          "5s",
-          "10s",
-          "30s",
-          "1m",
-          "5m",
-          "15m",
-          "30m",
-          "1h",
-          "2h",
-          "1d"
-        ],
-        "time_options": [
-          "5m",
-          "15m",
-          "1h",
-          "6h",
-          "12h",
-          "24h",
-          "2d",
-          "7d",
-          "28d"
-        ]
-      },
-      "timezone": "",
-      "title": "Image Builder Worker",
-      "uid": "image-builder-worker",
-      "version": 6,
-      "weekStart": ""
-    }
+      "time_options": [
+        "5m",
+        "15m",
+        "1h",
+        "6h",
+        "12h",
+        "24h",
+        "2d",
+        "7d",
+        "28d"
+      ]
+    },
+    "timezone": "",
+    "title": "Image Builder Worker with Tenant",
+    "uid": "JLSCLdqnk",
```

Comment:
Keep the `image-builder-worker` uid here; it determines the url of the dashboard.